qgreenland/config/layers.yml

```diff
@@ -482,20 +482,78 @@
 # Hydrology #
 #############
 
-- id: hydrologic_subbasins
-  title: 'Hydrologic Sub-Basins'
+- &streams_outlets_basins
+  id: ice_basins
+  title: 'Ice Basins'
   description: >-
-      Polygons of ice sheet hydrologic sub-basins with area in square
-      kilometers.
-  visible: False
-  group_path: 'Hydrology'
-  extent: 'data'
-  data_source: hydrologic_subbasins.only
-  ingest_task: rarred_shapefile
+      TODO
+  data_source: streams_outlets_basins.ice_basins
+  ingest_task: online_vector
   file_type: '.shp'
   data_type: 'vector'
+  visible: False
+  group_path: 'Hydrology'
+  ogr2ogr_kwargs:
+    input_filename: 'basins.gpkg'
+    makevalid: ''
```

Comment:
This argument doesn't actually get passed to ogr2ogr, iirc. Maybe we should rethink how it's passed?